builder/vmware/common/step_register.go

```diff
@@ -29,7 +29,7 @@ func (s *StepRegister) Run(ctx context.Context, state multistep.StateBag) multis
 	if remoteDriver, ok := driver.(RemoteDriver); ok {
 		ui.Say("Registering remote VM...")
 		if err := remoteDriver.Register(vmxPath); err != nil {
-			err := fmt.Errorf("Error registering VM: %s", err)
+			err := fmt.Errorf("error registering vm: %s", err)
```

Comment:
Capitalisation fix here

```suggestion
			err := fmt.Errorf("error registering VM: %s", err)
```

Reply:
Resolved in 67054cd.